.env

```diff
@@ -0,0 +1,27 @@
+_APP_ENV=development
```

Comment:
Anyway for to reduce this list to bare-minimum? Stuff like `_APP_ENV` already default to best practice production value. I would keep here only *must have* settings. Same for the `.env` file example in the readme.

Reply:
I don't see anything beside _APP_ENV can be removed?
1. can't remove DB related env,
2. Better not to remove SMTP related stuff
3. May be can we remove Storage limit?
4. Functions env?

Reply:
Actually you can remove almost everything. Any setting that can already be set in the dockerfile itself can be removed from here. Leave only stuff that the user is likely to want and change (or must change).

For example, no real need in the Cloud Functions stuff for a quick setup. _APP_SYSTEM_RESPONSE_FORMAT is likely to not change on first setup. Storage and abuse limits as well. Also avoid any Appwrite emails, this gets our inbox busy.